Source/Core/Core/HW/WII_IPC.h

```diff
@@ -35,6 +36,36 @@ enum StarletInterruptCause
   INT_CAUSE_IPC_STARLET = 0x80000000
 };
 
+enum class GPIO : u32
+{
+  POWER = 0x1,
+  SHUTDOWN = 0x2,
+  FAN = 0x4,
+  DC_DC = 0x8,
+  DI_SPIN = 0x10,
+  SLOT_LED = 0x20,
+  EJECT_BTN = 0x40,
+  SLOT_IN = 0x80,
+  SENSOR_BAR = 0x100,
+  DO_EJECT = 0x200,
+  EEP_CS = 0x400,
+  EEP_CLK = 0x800,
+  EEP_MOSI = 0x1000,
+  EEP_MISO = 0x2000,
+  AVE_SCL = 0x4000,
+  AVE_SDA = 0x8000,
+  DEBUG0 = 0x10000,
+  DEBUG1 = 0x20000,
+  DEBUG2 = 0x40000,
+  DEBUG3 = 0x80000,
+  DEBUG4 = 0x100000,
+  DEBUG5 = 0x200000,
+  DEBUG6 = 0x400000,
+  DEBUG7 = 0x800000,
+};
+
+extern Common::Flags<GPIO> gpio_out;
```

Comment:
The `Flags` thing already somewhat acts as getters and setters, and I don't know if it would be possible to do the same thing if I had getter/setter functions that returned the `Flags` on top of it.  (I also don't think reading it with MMIO makes sense for it here, since these represent actual pins on the board that physically control devices, instead of memory locations accessed by IOS; these are the GPIOs set by broadway and not the ones written by starlet).

I could create a function like this:

```C++
bool IsGPIOActive(GPIO gpio) {
    return gpio_out[gpio];
}
```

but that feels rather silly.

Reply:
In that case, please put a `g_` prefix in the name of this variable (and the RTC flag one).